src/Microsoft.Graph/Models/Extensions/ContentConverterExtensions.cs

```diff
@@ -0,0 +1,62 @@
+﻿// ------------------------------------------------------------------------------
+//  Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the MIT License.  See License in the project root for license information.
+// ------------------------------------------------------------------------------
+
+using System;
+
+namespace Microsoft.Graph.Extensions
+{
+    /// <summary>
+    /// Implements Content Converter Extensions
+    /// </summary>
+    public static class ContentConverterExtensions
+    {
+        /// <summary>
+        /// Converts DateTimeTimeZone which is a Complex Type to DateTime
+        /// </summary>
+        /// <param name="dateTimeTimeZone"></param>
+        /// <returns></returns>
+        public static DateTime ToDateTime(this DateTimeTimeZone dateTimeTimeZone)
+        {
+            DateTime dateTime = DateTime.Parse(dateTimeTimeZone.DateTime);
```

Comment:
What date format are we expecting to be passed to Parse?  Will this be impacted by the user's OS settings?  Would it not make more sense to use DateTime.ParseExact ?

Reply:
@HRook do you have any insights on whether all DateTime passed via the graph Complex Type DateTimeTimeZone are always in the same date format?